src/commands/force/org/beta/create.ts

```diff
@@ -262,9 +261,9 @@ export class Create extends SfdxCommand {
   }
 
   private async setAliasAndDefaultUsername(username: string): Promise<void> {
-    const aliases = await Aliases.create(Aliases.getDefaultOptions());
+    const stateAggregator = await StateAggregator.getInstance();
     if (this.flags.setalias) {
-      await aliases.updateValue(this.flags.setalias, username);
+      stateAggregator.aliases.set(this.flags.setalias, username);
```

Comment:
I wrote this code and I was forced to use the `updateValue` function, Im glad `set` works just as well